doc/ubuntu_release_process.md

```diff
@@ -205,12 +197,35 @@ We have daily packaging recipes that upload to the [daily ppa](https://code.laun
 
 ### When the daily recipe build fails ###
 
-The daily recipe for each release checks out master and then merges the ubuntu/$release
-and builds the package from there.  From time to time, the patches in the
-ubuntu/$release branch need to be refreshed/updated as upstream/master changes.
+The daily recipe for each release checks out tip of master, merges the ubuntu/$release
+and builds the package from there.  Daily recipe failures occur under two conditions:
+ 1. Tip of master drifts enough that patches in the ubuntu/$release branch need to be refreshed/updated/dropped as upstream/master changes.
+ 1. You just finished a [cherry-pick an upstream commit](#cherry-pick-process) into an
+    ubuntu/$release branch and need to drop that cherry pick patch so daily package
+    recipes continue to build.
+
+#### Dropping a cherry-pick patch
+
+ * **During Ubuntu series feature freeze** we cannot use `new-upstream-snapshot` because it pulls in all upstream changes and we don't want to include other upstream commits if we need a second cherry pick during feature freeze:
+   1. LOCAL_PATCH_COMMIT=`git log --pretty=oneline debian/ | awk 'NR == 1{print $1}'`
```

Comment:
The formatting of these lines will be quite confusing; can we either replace this numbered list with a code block or ensure that the whole of each line is marked up appropriately?

Reply:
Is this first command run immediately after running cherry-pick?  If so, let's put it all together in a single shell session so we can see the flow of the user-story here.